### ISSUE TITLE
Info disappearing. 

### ISSUE THREAD

**Issue by Elisagig** [Mergpijp/GlobalOrthodoxy#156]
“On page ii an image of the crucifixion of Jesus Christ and on page iii a picture of the teacher of the translator to whom the book is dedicated besides other unmentioned teachers of the translator.”

After Habtom saved this, he cannot see this information. It is on "Image Title, Filecategory and Image Contents"

**Comment by Elisagig** [comment]
Also related issue: 
Colleagues can't see image content after saving
![image](https://user-images.githubusercontent.com/61408472/120785517-7f027600-c52d-11eb-9dd8-562ab034826f.png)

**Comment by Mergpijp** [comment]
Its not entirely clear maybe you can demonstrate this on friday when we have a meeting?

**Comment by Mergpijp** [comment]
Hmm so when creating the uploaded file you can go to authors and see the authors there? It seems to be working from here

**Comment by Elisagig** [comment]
Just asked Habtom. He said he does not remember this issue but he says that "edit Author, Files(images)" gives still error after submit"

**Comment by Mergpijp** [comment]
Yes but that is related to another issue @Elisagig . #176 and should be solved now.

**Comment by Woseseltops** [comment]
For your information, @Elisagig , #176 is now live (hotfix) so you can check if this issue is solved now.

**Comment by Elisagig** [comment]
Thanks. Will ask my colleagues
